libobjrenderer/src/objectsscene.cpp

```diff
@@ -1433,6 +1441,14 @@ bool ObjectsScene::isMovingObjects(void)
 	return(moving_objs);
 }
 
+void ObjectsScene::selectItem(BaseObjectView * item)
```

Comment:
Why create a method that selects an item? All scene items has the setSelected that triggers the needed signals to handle item selection.

Reply:
ahem... that is why I said "might not be needed" :fearful: :D
You can put it to garbage.